src/views/InteractiveMap/InteractiveMap.vue

```diff
@@ -255,6 +256,10 @@ export default Vue.extend({
           this.hasLoadMapError = true;
         });
     },
+    addEmptyCard(dataDriven) {
+      const name = dataDriven ? "Data driven" : "Design";
+      this.addCard(name, null, null, "", dataDriven);
```

Comment:
Since `method` can be unset now, it would be nice with a default text label in the card (something like "Not selected"), like we have for organism and model.

![image](https://user-images.githubusercontent.com/64745/58472075-b9b6c400-8145-11e9-81ba-72c244ffed9a.png)

Reply:
Sorry for not thinking this through, I think method should be preselected as pfba.